src/menu_items.c

```diff
@@ -3738,35 +3734,7 @@ void clear_menu_textures(void) {
     sMenuTextureEntries = 0;
 }
 
-/**
- * Differs from memory.c with `+ 0x8` instead of `| 0x8`
- *
- * @param addr
- * @return void*
- */
-void* segmented_to_virtual_dupe(const void* addr) {
-#ifdef TARGET_N64
-    size_t segment = (uintptr_t) addr >> 24;
-    size_t offset = (uintptr_t) addr & 0x00FFFFFF;
-
-    return (void*) ((gSegmentTable[segment] + offset) + 0x80000000);
-#else
-    return addr;
-#endif
-}
-
-void* segmented_to_virtual_dupe_2(const void* addr) {
-#ifdef TARGET_N64
-    size_t segment = (uintptr_t) addr >> 24;
-    size_t offset = (uintptr_t) addr & 0x00FFFFFF;
-
-    return (void*) ((gSegmentTable[segment] + offset) + 0x80000000);
-#else
-    return addr;
-#endif
-}
-
-#include <assets/player_selection.h>
+#include <assets/textures/player_selection.h>
```

Comment:
Maybe move this include to the top of the file. Unless there's a reason to leave it here?

Reply:
don't really touch it but we can fix it for another pr